frontend/src/pages/Import.tsx

```
@@ -0,0 +1,143 @@
+        <p style={{ margin: 0 }}><b>CSV Format:</b> <code>provider,amount,currency,dueISO,autopay</code></p>
+        <p style={{ margin: '0.5rem 0 0', fontSize: '0.9rem', color: '#666' }}>Simple comma-delimited. No quotes or commas in values.</p>
+      </div>
+      <div style={s.drop}>
+        <p>Drag CSV file here or choose file</p>
+        <input type="file" accept=".csv,text/csv" onChange={handleFileChange} />
+      </div>
+      {file && (
+        <div style={{ marginBottom: '1rem' }}>
+          <p>Selected: {file.name}</p>
+          <button onClick={handleProcessCSV} disabled={processing}>
+            {processing ? 'Processing...' : 'Process CSV'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)